web/themes/custom/fmcg_material/sass/_button.scss

```diff
@@ -0,0 +1,47 @@
+input.button, .btn, a.btn, span.btn, a.btn.pull-right, .btn-pull-right, input[type="submit"] {
+  height: 36px;
+  min-width: 88px;
+  margin: 8px;
+  vertical-align: middle;
+  border-radius: 2px;
+  color: $fmcg-black;
+  background: $fmcg-yellow;
+  cursor: pointer;
+  display: inline-block;
+  font-family: Poppins;
+  font-size: 14px;
+  font-weight: 600;
+  letter-spacing: 0.5px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition: color 0.15s cubic-bezier(0.4, 0, 0.2, 1), background-color 0.15s cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Look up browser compatibility to transition. I think its one of those things where you want -webkit-transition and another one as well - just in case.